README.md

```diff
@@ -13,6 +13,18 @@ To automate the administration of `greenelab/scrum` issues, this repository reli
 Every day, Travis CI executes the commands in [`.travis.yml`](.travis.yml).
 As appropriate, **@scrum-lord** closes and opens issues to keep the scrum issues up to date.
 
+## Deployment Instructions
+
+1. Fork repo
+1. Create new empty repo organization/scrum
+1. Get Github login token (https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) with repo scope
```

Comment:
It's probably worth mentioning that the safest way to do this is to create a new machine user that doesn't have any other privileges than for the scrum repo. In general, I'm a bit hesitant to give a CI build token access to all of my personal account repos. This is why we made @scrum-lord.